frontend/src/features/admin-form/create/builder-and-design/MagicFormBuilderModal/MagicFormBuilderModalContent/PdfDetailsScreen.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react'
+import { Controller } from 'react-hook-form'
+import { BiRightArrowAlt } from 'react-icons/bi'
+import {
+  Container,
+  Flex,
+  FormControl,
+  FormErrorIcon,
+  FormErrorMessage,
+  FormLabel,
+  ModalBody,
+  ModalHeader,
+  Text,
+} from '@chakra-ui/react'
+import pdfjsWorker from 'pdfjs-dist/build/pdf.worker.entry'
+import * as pdfjs from 'pdfjs-dist/legacy/build/pdf'
+
+import { MB } from '~shared/constants'
+
+import { BxsErrorCircle } from '~assets/icons'
+import Button from '~components/Button'
+import Attachment from '~components/Field/Attachment'
+
+import { useMagicFormBuilderWizard } from '../MagicFormBuilderWizardContext'
+
+export const MagicFormBuilderPdfDetailsScreen = (): JSX.Element => {
+  const {
+    handleDetailsSubmit,
+    isLoading,
+    isFetching,
+    handleBack,
+    formMethods,
+  } = useMagicFormBuilderWizard()
+
+  const {
+    setError,
+    control,
+    setValue,
+    formState: { errors },
+  } = formMethods
+
+  const MAX_FILE_SIZE = 5 * MB
+  const VALID_EXTENSIONS = '.pdf'
```

Comment:
These are constants which are magic values. Should we factor them out to a `constants.ts` file?